### PR TITLE
Windows dotnet web application

### PR DESCRIPTION
**Pull request by michaeldigiacomi**
*Description of changes:*

Updated the repository to include a Windows Server Docker Example for Image Builder

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CloudFormation/Docker/windows-dotnet-web-application/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+Next, update the CloudFormation parameters .json file (```windows-dotnet-web-application-pipeline.json```) with the S3 object used in the AWS CLI command.
+
+### Deploying the Stack
+
+#### AWS Management Console
+
+1. Upload the ```windows-dotnet-web-application-pipeline.yml``` template to CloudFormation.
+2. Update the stack parameters as desired, ensuring the ```DotnetS3SourceTarFile``` parameter points to the S3 location used when uploading the .NET web application to S3.
```
[the remark this answers is not in the document]
Comment (michaeldigiacomi) [reply CloudFormation/Docker/windows-dotnet-web-application/README.md]: I have updated this as well as the documentation to zip the file.

File: CloudFormation/Docker/windows-dotnet-web-application/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Windows Server 2019 Container Image Pipeline for hosting a .NET web application
+
+This is a sample template that demonstrates how to use EC2 Image Builder CloudFormation resources to build an Windows Server 2019 Docker container image that can host a .NET web application. The image will be published to the specified Amazon Elastic Container Registry (ECR) repository.
```
Comment (austoonz) [inline CloudFormation/Docker/windows-dotnet-web-application/README.md:3]: `an` -> `a`: "...to build a Windows Server..."

(only commented on this as there's a tag that should be updated as well)

File: CloudFormation/Docker/windows-dotnet-web-application/windows-dotnet-web-application-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+        ImageTestsEnabled: false
+        TimeoutMinutes: 60
+      InfrastructureConfigurationArn: !Ref Infrastructure
+      Name: !Ref ResourceName
+      Schedule:
+        PipelineExecutionStartCondition: EXPRESSION_MATCH_AND_DEPENDENCY_UPDATES_AVAILABLE
+        ScheduleExpression: 'cron(0 10 * * ? *)'
+      Status: ENABLED
+      Tags:
+        Purpose: PowerShellSummitContainerDemo
```
Comment (austoonz) [inline CloudFormation/Docker/windows-dotnet-web-application/windows-dotnet-web-application-pipeline.yml:295]: We should update the tag for this sample.

**Review by jterry75 — APPROVED** [review]
LGTM

**Review by austoonz — CHANGES_REQUESTED** [review]
Looks good! Two minor comments (the first nit only because there's also a tag we should update before merging).

File: CloudFormation/Docker/windows-dotnet-web-application/windows-dotnet-web-application-pipeline.yml (hunk 3 above)
Comment (michaeldigiacomi) [reply CloudFormation/Docker/windows-dotnet-web-application/windows-dotnet-web-application-pipeline.yml]: Tag has been dropped.

**Review by michaeldigiacomi — COMMENTED** [review]
Makes sense

**Pull request merged**